validation/src/test/scala/hmda/validation/rules/lar/quality/Q013Spec.scala

```diff
@@ -0,0 +1,40 @@
+package hmda.validation.rules.lar.quality
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q013Spec extends LarEditCheckSpec with BadValueUtils {
+
+  val config = ConfigFactory.load()
+  val minAmount = config.getInt("hmda.validation.quality.Q013.loan.min-amount")
+  val maxAmount = config.getInt("hmda.validation.quality.Q013.loan.max-amount")
+
+  property("Valid if property type not 3") {
+    forAll(larGen, intOtherThan(3)) { (lar, x) =>
+      val newLoan = lar.loan.copy(propertyType = x)
+      val newLar = lar.copy(loan = newLoan)
+      newLar.mustPass
+    }
+  }
+
+  property("Valid if loan amount between 100 ($100,000) and 10000 ($10,000,000)") {
```

Comment:
I feel like these should be parameterized.  If they are going to change in the future, it would be nice to not have to change the property messages as well.

`s"Valid if loan amount between $minAmount and $maxAmount"`